### DISCUSSION TITLE
Is it possible to select a template on the fly during capture or template creation?

### DISCUSSION THREAD

**Discussion by billkach** [chhoumann/quickadd#561] [Q&A]
I'm wanting to see functionality where, just the same as I can configure QuickAdd to allow me to select the folder for my new capture or template to end up in on the fly, it will allow me to select which template I want to use for the new file during the configuration, rather than having to configure a single QuickAdd item for each template. Is this functionality already present and I'm just missing it?

**Comment by chhoumann** [comment]
Very late answer - sorry about that. This shipped since, in two flavors:

1. **New note from template** - a built-in command (and a row in the Run QuickAdd launcher) that lists every template in your configured template folder, prompts for the note name, and creates it. No per-template choice needed; you just need a template folder set in QuickAdd's settings.

2. **Format syntax in the Template Path** - keep a single Template choice and make its path dynamic:

```
Templates/{{VALUE:collectionName}} Template.md
```

Each run prompts for the value and resolves to the matching template file. The same works in a Capture choice's *Create file with template* path, and it combines with the on-the-fly folder selection you're already using.

Docs: https://quickadd.obsidian.guide/docs/Choices/TemplateChoice

Closing as resolved.
